Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-slim
+RUN useradd -ms /bin/bash newrelic-cli
```

Comment:
We probably want to use `newrelic-lambda-cli` to be consistent and to avoid any possible future naming conflicts.